apps/condo/domains/notification/schema/SendMessageService.js

```diff
@@ -152,7 +172,7 @@ const SendMessageService = new GQLCustomSchema('SendMessageService', {
                     ? messageWithSameUniqKey
                     : await Message.create(context, messageAttrs)
 
-                if (!messageWithSameUniqKey) await deliverMessage.delay(message.id)
+                if (!messageWithSameUniqKey) await deliverMessage.applyAsync([message.id], getMessageQueue({ type }))
```

Comment:
What about changing the `deliverMessage()` signature?

Reply:
`delay` takes args like so:
```
function delay(...args) {
   console.log(args)
}
```
`applyAsync` has the following signature:
```
function applyAsync(args, queue, bullOptions) {
    console.log(args)
}
```

So `delay(arg1, arg2, arg3)` is equal to `applyAsync([arg1, arg2, arg3])`, but `applyAsync` can take extra options, that's the main difference